client/my-sites/theme/main.jsx

```diff
@@ -130,10 +130,11 @@ const ThemeSheet = React.createClass( {
 	},
 
 	getValidSections() {
+		const { isCurrentUserPaid, isJetpack, forumUrl, isWpcomTheme, isLoggedIn } = this.props;
 		const validSections = [];
 		validSections.push( '' ); // Default section
 		this.props.supportDocumentation && validSections.push( 'setup' );
-		validSections.push( 'support' );
+		( ! isLoggedIn || ( isCurrentUserPaid && ! isJetpack ) || forumUrl || isWpcomTheme ) && validSections.push( 'support' );
```

Comment:
> ( ! isLoggedIn || ( isCurrentUserPaid && ! isJetpack ) || forumUrl || isWpcomTheme )

This logic seems a little fragile. Any time someone makes a tweak to the support UI logic below, they would have to know to change this line as well. Also this won't stop inbound links to /support.

Maybe we could have a default support message when none of the other cases apply?